Susi/Controllers/SkillListingViewController/SkillListingViewController.swift

```diff
@@ -26,6 +26,17 @@ class SkillListingViewController: UITableViewController {
         return ib
     }()
 
+    lazy var languageButton: UIButton = {
+        let button  = UIButton()
+        button.backgroundColor = UIColor.white
+        button.layer.shadowColor = UIColor.blue.cgColor
+        button.layer.cornerRadius = 18.0
+        button.setTitleColor(UIColor.defaultColor(), for: .normal)
+        button.addTarget(self, action: #selector(getToAllLanguageVC), for: .touchUpInside)
+        button.setTitle("Language", for: .normal)
+        return button
+    }()
+    
```

Comment:
Can we keep the background same as navigation bar (Blue) and text in white color rather than making it a white background and blue color? I think that that would be good from UI perspective.  Also, we can just add the bottom border (not full border).

Reply:
Done